apps/reviews/templates/reviews/review.html

```diff
@@ -83,16 +83,31 @@ <h3 class="addon-name"><a href="{{ addon.get_url_path() }}">{{ addon.name }}</a>
       </li>
     {% endif %}
     {% if review.user_id == request.user.id %}
-      <li>
-        <a class="review-edit" href="#">
-          {{ _('Edit review') }}</a>
-      </li>
+      {% if is_reply %}
+        <li>
+          <a class="review-edit" href="#">
+            {{ _('Edit reply') }}</a>
+        </li>
+      {% elif not is_reply %}
```

Comment:
You simply need to write `{% else %}` here, it's a boolean case, there's no other choice.